sql/builtin_functions.cc

```diff
@@ -3275,6 +3275,24 @@ BoundFunction fetcher(const std::vector<BoundSqlExpression> & args)
 }
 static RegisterBuiltin registerFetcherFunction(fetcher, "fetcher");
 
+BoundFunction isconst(const std::vector<BoundSqlExpression> & args)
```

Comment:
should be static_is_constant() to make it clear that we're testing the static analysis